cirq-core/cirq/study/result.py

```diff
@@ -80,52 +82,35 @@ def _key_to_str(key: TMeasurementKey) -> str:
     return ','.join(str(q) for q in key)
 
 
-class Result:
+class Result(abc.ABC):
```

Comment:
Update class docstring to reflect new behavior.

Reply:
Done.